documentation/src/main/asciidoc/topics/proc_marshalling_external_classes.adoc

```diff
@@ -0,0 +1,41 @@
+[id='implementing_proto_marshallers']
+= Marshalling External Classes
+If you want to use external, third-party Java object classes, but cannot add annotations to them, create adapter classes instead.
```

Comment:
If the objects are third-party classes, then the user won't be able to annotate them. So maybe just:

"If you want to use external, third-party Java object classes, it's necessary to create an adapter class."

Reply:
@oraNod This file seems to be missing from the latest commit

Reply:
@ryanemerson yes, I thought that was the idea. combine the procedure for user types as well as external classes so there is one SCI.